File: TechTest/app/src/main/java/com/david_kapas/android/techtest/logic/api/model/Post.java
package com.david_kapas.android.techtest.logic.api.model;

import android.arch.persistence.room.Entity;
import android.support.annotation.NonNull;

/**
 * Pojo class Post object.
 * Created by David Kapas on 2018.03.17.
 */
@Entity(primaryKeys = {"id"})
public class Post {

    private int userId;
    @NonNull
    private int id;
    private String title;
    private String body;

    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }
}

File: TechTest/app/src/main/java/com/david_kapas/android/techtest/logic/api/model/Post.kt
package com.david_kapas.android.techtest.logic.api.model

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

/**
 * Pojo class Post object.
 * Created by David Kapas on 3/8/2019.
 */
@Entity
data class Post(@PrimaryKey var id: Int?,
                @ColumnInfo(name = "userId") var userId: Int,
                @ColumnInfo(name = "title") var title: String?,
                @ColumnInfo(name = "body") var body: String?) {
    constructor() : this(null, 0, "", "");
}